pom.xml

```diff
@@ -78,6 +78,7 @@
         <sonar.host.url>https://sonarcloud.io</sonar.host.url>
 
         <jdbi.version>2.78</jdbi.version>
+        <org.reflections.version>0.10.2</org.reflections.version>
```

Comment:
Need `reflections` for helping users take in a list of packages and automagically return all "Dao" specific classes annotated with `SqlUpdate` / `SqlQuery` / `SqlCall` / SqlBatch` etc.